SketchUp/Sketchup/ComponentInstance.rb

```diff
@@ -477,8 +477,8 @@ def split(instance)
   def subtract(instance)
   end
 
-  # The transform! method is used to apply a transformation to a component
-  # instance.
+  # The +transform!+ method is used to apply a {Geom::Transformation} to a
```

Comment:
And on that topic I'm never been too keen on the "The method_name is doing x" way of introducing a method, I feel is echoing too much. However, it's used so consistently across the 1000+ methods already documented that there might not be any point trying to change that now?